test/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ FetchContent_MakeAvailable(googletest)
 enable_testing()
 
 set(UMF_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(UMF_UTILS_DIR ${UMF_CMAKE_SOURCE_DIR}/src/utils)
```

Comment:
I'm not sure how this should be done but why you don't use CMAKE_CURRENT_SOURCE_DIR as in the line above?

Reply:
Because `CMAKE_CURRENT_SOURCE_DIR` refers to the **current** source directory, while `UMF_CMAKE_SOURCE_DIR` refers to the **main source directory**.